src/test/java/org/prebid/server/ApplicationTest.java

```diff
@@ -111,73 +115,209 @@ public static void setUp() throws IOException {
+        final Response response = given(spec)
+                .header("Referer", "http://www.example.com")
+                .header("X-Forwarded-For", "192.168.244.1")
+                .header("User-Agent", "userAgent")
+                .header("Origin", "http://www.example.com")
+// 		           this uids cookie value stands for
+//                {"uids":{"conversant":"CV-UID"}}
+                .cookie("uids", "eyJ1aWRzIjp7ImNvbnZlcnNhbnQiOiJDVi1VSUQifX0=")
+                .body(jsonFrom("openrtb2/conversant/test-auction-conversant-request.json"))
+                .post("/openrtb2/auction");
 
+        //then
+        final Map<String, String> exchanges = new HashMap<>();
+        exchanges.put(CONVERSANT, "http://localhost:" + WIREMOCK_PORT + "/conversant-exchange");
+        exchanges.put(CONVERSANT_ALIAS, null);
+
+        final String expectedAuctionResponse = auctionResponseFrom(jsonFrom("openrtb2/conversant/test-auction-conversant-response.json"),
+                response, OPENRTB_RESPONSE_TIME_PLACEHOLDER, exchanges);
+
+        JSONAssert.assertEquals(expectedAuctionResponse, response.asString(), JSONCompareMode.NON_EXTENSIBLE);
+    }
+
+    @Test
+    public void openrtb2AuctionShouldRespondWithBidsFromIx() throws IOException, JSONException {
+        //given
         // ix bid response for imp 6 and imp 61
         wireMockRule.stubFor(post(urlPathEqualTo("/ix-exchange"))
-                .withRequestBody(equalToJson(jsonFrom("openrtb2/test-ix-bid-request-1.json")))
-                .willReturn(aResponse().withBody(jsonFrom("openrtb2/test-ix-bid-response-1.json"))));
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/ix/test-ix-bid-request-1.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/ix/test-ix-bid-response-1.json"))));
+
+        // pre-bid cache
+        wireMockRule.stubFor(post(urlPathEqualTo("/cache"))
+                .withRequestBody(equalToJson(jsonFrom("openrtb2/ix/test-cache-ix-request.json")))
+                .willReturn(aResponse().withBody(jsonFrom("openrtb2/ix/test-cache-ix-response.json"))));
+
+        //when
+        final Response response = given(spec)
+                .header("Referer", "http://www.example.com")
+                .header("X-Forwarded-For", "192.168.244.1")
+                .header("User-Agent", "userAgent")
+                .header("Origin", "http://www.example.com")
+// 		           this uids cookie value stands for
+//                {"uids":{"ix":"IE-UID"}}
+                .cookie("uids", "eyJ1aWRzIjp7Iml4IjoiSUUtVUlEIn19")
+                .body(jsonFrom("openrtb2/ix/test-auction-ix-request.json"))
+                .post("/openrtb2/auction");
+
+        //then
+        final String expectedAuctionResponse = auctionResponseFrom(jsonFrom("openrtb2/ix/test-auction-ix-response.json"),
+                response, OPENRTB_RESPONSE_TIME_PLACEHOLDER, Collections.singletonMap(IX, "http://localhost:" + WIREMOCK_PORT + "/ix-exchange"));
```

Comment:
In test it is allowed and advised to use static imports for methods like `singletonMap`